cljfmt/src/cljfmt/main.clj

```diff
@@ -34,8 +34,7 @@
       [f])))
 
 (defn- reformat-string [options s]
-  (cljfmt/reformat-string s options))
-
+  ((cljfmt/wrap-normalize-newlines #(cljfmt/reformat-string % options)) s))
```

Comment:
Can you put the missing newline back in?

Reply:
👍 sorry didn't clock it last night. updating now